packages/genesys-spark-components/src/components/stable/gux-time-picker/tests/gux-time-picker.spec.ts

```diff
@@ -1,18 +1,71 @@
-import { newSpecPage } from '@test/specTestUtils';
+const useRegionalDates = jest.fn().mockReturnValue(false);
+
+jest.mock('i18n/use-regional-dates', () => ({
+  __esModule: true,
+  useRegionalDates
+}));
 
+import { newSpecPage } from '@test/specTestUtils';
 import { GuxTimePicker } from '../gux-time-picker';
 
 const components = [GuxTimePicker];
 const html = `<gux-time-picker value="09:00"></gux-time-picker>`;
-const language = 'en';
+const languageList = [
```

Comment:
It might be worth moving the language array to the test utilities. Could be useful in other tests and would make updating for new localizations slightly easier. Maybe not.